Misc/NEWS.d/next/Tests/2025-04-26-11-34-07.gh-issue-91221.qlhMVS.rst

```diff
@@ -0,0 +1,3 @@
+Do not check the output of :func:`curses.pair_content(0)
```

Comment:
We usually do not add NEWS entries for test-only changes. They are not useful for Python users.